commit 0c8a30478e21598b6445c174024deb210bde8628
Author: Elvis de Souza <elvis.desouza99@gmail.com>
Date:   Tue Jul 13 19:50:12 2021 -0300

    "se" seguido de particípio
    
    "se" SCONJ/mark
    
    "se" provavelmente será SCONJ/mark se dependente de token não verbal (e não auxiliar) ou se VerbForm=Part

diff --git a/Petroles_3/Petroles_3_golden.conllu b/Petroles_3/Petroles_3_golden.conllu
index a5d1aaf..fd67390 100644
--- a/Petroles_3/Petroles_3_golden.conllu
+++ b/Petroles_3/Petroles_3_golden.conllu
@@ -19186,7 +19186,7 @@
 13	tratado	tratar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	17	advcl	O	_
 14	,	,	PUNCT	_	_	13	punct	O	_
 15	tenha	ter	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	17	aux	O	_
-16	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	17	expl	O	_
+16	se	se	PRON	_	_	17	expl	O	_
 17	deteriorado	deteriorar	VERB	_	VerbForm=Part	1	ccomp	O	_
 18	,	,	PUNCT	_	_	19	punct	O	_
 19	ocasionando	ocasionar	VERB	_	VerbForm=Ger	17	advcl	O	_
@@ -24416,7 +24416,7 @@
 # sent_id = 284-20150917-MONOGRAFIA_0-151
 1	Dentre	dentre	ADP	_	_	3	case	O	_
 2	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	3	det	O	_
-3	trabalhos	trabalho	NOUN	_	Gender=Masc|Number=Plur	18	nmod	O	_
+3	trabalhos	trabalho	NOUN	_	Gender=Masc|Number=Plur	16	obl	O	_
 4	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	5	nsubj	O	_
 5	consideram	considerar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	3	acl:relcl	O	_
 6	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	7	det	O	_
@@ -24429,18 +24429,18 @@
 12	Romualdo	Romualdo	PROPN	_	Number=Sing	11	flat:name	O	_
 13	como	como	ADP	_	_	14	case	O	_
 14	marinho	marinho	NOUN	_	Gender=Masc|Number=Sing	5	xcomp	O	_
-15	,	,	PUNCT	_	_	5	punct	O	_
+15	,	,	PUNCT	_	_	3	punct	O	_
 16-17	tem-se	_	_	_	_	_	_	_	_
-16	tem	ter	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	18	aux	O	_
-17	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	16	expl	O	_
-18	o	o	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	0	root	O	_
+16	tem	ter	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
+17	se	se	PRON	_	_	16	expl	O	_
+18	o	o	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	16	obj	O	_
 19	de	de	ADP	_	_	20	case	O	_
 20	Beurlen	Beurlen	PROPN	_	Gender=Masc|Number=Sing	18	nmod	O	_
 21	(	(	PUNCT	_	_	22	punct	O	_
-22	1966	1966	NUM	_	NumType=Card	20	nmod:appos	O	_
+22	1966	1966	NUM	_	NumType=Card	18	nmod:appos	O	_
 23	)	)	PUNCT	_	_	22	punct	O	_
 24	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	25	nsubj	O	_
-25	descreve	descrever	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	20	acl:relcl	O	_
+25	descreve	descrever	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	18	acl:relcl	O	_
 26	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	28	det	O	_
 27	primeiros	primeiro	ADJ	_	Gender=Masc|NumType=Ord|Number=Plur	28	amod	O	_
 28	organismos	organismo	NOUN	_	Gender=Masc|Number=Plur	25	obj	O	_
@@ -24460,7 +24460,7 @@
 41	Pygurus	Pygurus	PROPN	_	Gender=Masc|Number=Sing	38	conj	O	_
 42	tinocoi	tinocoi	PROPN	_	Number=Sing	41	flat:name	O	_
 43	)	)	PUNCT	_	_	38	punct	O	_
-44	,	,	PUNCT	_	_	35	punct	O	_
+44	,	,	PUNCT	_	_	36	punct	O	_
 45	que	que	PRON	_	Gender=Masc|Number=Plur|PronType=Rel	47	nsubj	O	_
 46	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	47	expl	O	_
 47	assemelham	assemelhar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	36	acl:relcl	O	_
@@ -24487,7 +24487,7 @@
 65-66	na	_	_	_	_	_	_	_	_
 65	em	em	ADP	_	_	67	case	O	_
 66	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	67	det	O	_
-67	região	região	NOUN	_	Gender=Fem|Number=Sing	61	obl	O	_
+67	região	região	NOUN	_	Gender=Fem|Number=Sing	63	nmod	O	_
 68-69	do	_	_	_	_	_	_	_	_
 68	de	de	ADP	_	_	70	case	O	_
 69	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	70	det	O	_
@@ -24495,13 +24495,13 @@
 71-72	pelo	_	_	_	_	_	_	_	_
 71	por	por	ADP	_	_	73	case	O	_
 72	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	73	det	O	_
-73	norte	norte	NOUN	_	Gender=Masc|Number=Sing	67	nmod	O	_
-74	,	,	PUNCT	_	_	61	punct	O	_
+73	norte	norte	NOUN	_	Gender=Masc|Number=Sing	63	nmod	O	_
+74	,	,	PUNCT	_	_	78	punct	O	_
 75	através	através	ADV	_	_	78	case	O	MWEPOS=ADP
 76-77	da	_	_	_	_	_	_	_	_
 76	de	de	ADP	_	_	75	fixed	O	_
 77	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	78	det	O	_
-78	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	61	obl	B=BACIA:TIPO	_
+78	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	63	nmod	B=BACIA:TIPO	_
 79-80	do	_	_	_	_	_	_	_	_
 79	de	de	ADP	_	_	78	flat:name	O	_
 80	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	78	flat:name	O	_
@@ -35311,11 +35311,11 @@
 37	comparado	comparar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	34	advcl	O	_
 38	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	39	det	O	_
 39	produção	produção	NOUN	_	Gender=Fem|Number=Sing	37	obj	O	_
-40	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	44	nsubj:pass	O	_
-41	seria	ser	AUX	_	Mood=Cnd|Number=Sing|Person=3|VerbForm=Fin	44	aux:pass	O	_
+40	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	41	nsubj	O	_
+41	seria	ser	VERB	_	Mood=Cnd|Number=Sing|Person=3|VerbForm=Fin	39	acl:relcl	O	_
 42	se	se	SCONJ	_	_	44	mark	O	_
 43	fosse	ser	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Imp|VerbForm=Fin	44	aux:pass	O	_
-44	utilizada	utilizar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	39	acl:relcl	O	_
+44	utilizada	utilizar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part|Voice=Pass	41	advcl	O	_
 45	apenas	apenas	ADV	_	_	47	advmod	O	_
 46	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	47	det	O	_
 47	energia	energia	NOUN	_	Gender=Fem|Number=Sing	44	nsubj:pass	O	_
@@ -38777,7 +38777,7 @@
 # text = Se for um caso variante no tempo, então e , assim , como consequência das equações de Maxwell..
 # sent_id = 398-20160721-MONOGRAFIA_0-188
 1	Se	se	SCONJ	_	_	4	mark	O	_
-2	for	ser	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	4	cop	O	_
+2	for	ser	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	4	cop	O	eliminar
 3	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	4	det	O	_
 4	caso	caso	NOUN	_	Gender=Masc|Number=Sing	0	root	O	_
 5	variante	variante	ADJ	_	Gender=Fem|Number=Sing	4	amod	O	_
@@ -61821,7 +61821,7 @@
 3	isso	isso	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	1	fixed	O	_
 4	,	,	PUNCT	_	_	5	punct	O	_
 5-6	observando-se	_	_	_	_	_	_	_	_
-5	observando	observar	VERB	_	VerbForm=Ger	39	advcl	O	_
+5	observando	observar	VERB	_	VerbForm=Ger	15	advcl	O	_
 6	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	5	expl	O	_
 7	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	8	det	O	_
 8	resultados	resultado	NOUN	_	Gender=Masc|Number=Plur	5	obj	O	_
@@ -61833,7 +61833,7 @@
 13	V.3	V.3	PROPN	_	Number=Sing	12	flat:name	O	_
 14	,	,	PUNCT	_	_	5	punct	O	_
 15-16	nota-se	_	_	_	_	_	_	_	_
-15	nota	nota	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
+15	nota	notar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 16	se	se	PRON	_	_	15	expl	O	_
 17	que	que	SCONJ	_	_	39	mark	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
@@ -61854,13 +61854,13 @@
 32	dialildimetilamônio	dialildimetilamônio	NOUN	_	Gender=Masc|Number=Sing	30	nmod	O	_
 33	)	)	PUNCT	_	_	30	punct	O	_
 34	como	como	ADP	_	_	35	case	O	_
-35	inibidor	inibidor	NOUN	_	Gender=Masc|Number=Sing	30	nmod	O	_
+35	inibidor	inibidor	NOUN	_	Gender=Masc|Number=Sing	25	obl	O	_
 36	,	,	PUNCT	_	_	25	punct	O	_
 37	foi	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	39	cop	O	_
 38	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	39	det	O	_
-39	argila	argila	NOUN	_	Gender=Fem|Number=Sing	0	root	B=LITOLOGIA	_
+39	argila	argila	NOUN	_	Gender=Fem|Number=Sing	15	ccomp	B=LITOLOGIA	_
 40	A4	A4	PROPN	_	Gender=Fem|Number=Sing	39	nmod	O	_
-41	..	..	PUNCT	_	_	39	punct	O	_
+41	..	..	PUNCT	_	_	15	punct	O	_
 
 # text = A argila A4 curiosamente é a mais reativa de todas as argilas estudadas..
 # sent_id = 24-20150122-MONOGRAFIA_0-559
@@ -64061,7 +64061,7 @@
 26	.	.	PUNCT	_	_	20	punct	O	_
 27			CCONJ	_	_	28	punct	O	_
 28-29	Observou-se	_	_	_	_	_	_	_	_
-28	Abservou	Abservou	PROPN	_	Gender=Masc|Number=Sing	20	nsubj	O	_
+28	Abservou	Abservou	PROPN	_	Gender=Masc|Number=Sing	20	nsubj	O	sentenciação
 29	se	se	PRON	_	_	28	expl	O	_
 30	que	que	SCONJ	_	_	41	mark	O	_
 31	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	33	det	O	_
@@ -79376,7 +79376,7 @@
 15	projeto	projeto	NOUN	_	Gender=Masc|Number=Sing	18	nsubj	O	_
 16	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	18	cop	O	_
 17	realmente	realmente	ADV	_	_	18	advmod	O	_
-18	capaz	capaz	ADJ	_	Gender=Masc|Number=Sing	2	acl:relcl	O	_
+18	capaz	capaz	ADJ	_	Gender=Masc|Number=Sing	5	conj	O	_
 19	de	de	SCONJ	_	_	20	mark	O	_
 20	mitigar	mitigar	VERB	_	VerbForm=Inf	18	xcomp	O	_
 21	emissões	emissão	NOUN	_	Gender=Fem|Number=Plur	20	obj	O	_
@@ -137606,14 +137606,14 @@
 
 # text = Toda vez que dois líquidos imiscíveis entre si, como óleo e água, são expostos a uma grande agitação, como por exemplo, ao longo do percurso do petróleo desde o reservatório até a superfície, o líquido em menor proporção, no caso a água, se dispersa no outro, gerando gotas de diversos diâmetros..
 # sent_id = 202-20140404-MONOGRAFIA_0-19
-1	Toda	todo	DET	_	Gender=Fem|Number=Sing|PronType=Tot	5	det	O	_
+1	Toda	todo	DET	_	Gender=Fem|Number=Sing|PronType=Tot	5	mark	O	MWEPOS=SCONJ
 2	vez	vez	ADP	_	_	1	fixed	O	_
 3	que	que	SCONJ	_	_	1	fixed	O	_
 4	dois	dois	NUM	_	NumType=Card	5	nummod	O	_
 5	líquidos	líquido	NOUN	_	Gender=Masc|Number=Plur	16	nsubj:pass	O	_
 6	imiscíveis	imiscível	ADJ	_	Gender=Masc|Number=Plur	5	amod	O	_
 7	entre	entre	ADP	_	_	8	case	O	_
-8	si	si	PRON	_	Gender=Masc|Number=Sing|Person=3|PronType=Prs	5	nmod	O	_
+8	si	si	PRON	_	Gender=Masc|Number=Sing|Person=3|PronType=Prs	6	obl	O	_
 9	,	,	PUNCT	_	_	11	punct	O	_
 10	como	como	ADP	_	_	11	case	O	_
 11	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	5	nmod:appos	B=FLUIDO	_
@@ -137621,7 +137621,7 @@
 13	água	água	NOUN	_	Gender=Fem|Number=Sing	11	conj	B=FLUIDO	_
 14	,	,	PUNCT	_	_	11	punct	O	_
 15	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	16	aux:pass	O	_
-16	expostos	expor	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
+16	expostos	expor	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	55	advcl	O	_
 17	a	a	ADP	_	_	20	case	O	_
 18	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
 19	grande	grande	ADJ	_	Gender=Fem|Number=Sing	20	amod	O	_
@@ -137649,9 +137649,9 @@
 38	até	até	ADP	_	_	40	case	O	_
 39	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	40	det	O	_
 40	superfície	superfície	NOUN	_	Gender=Fem|Number=Sing	37	nmod	O	_
-41	,	,	PUNCT	_	_	43	punct	O	_
+41	,	,	PUNCT	_	_	28	punct	O	_
 42	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	43	det	O	_
-43	líquido	líquido	NOUN	_	Gender=Masc|Number=Sing	40	conj	O	_
+43	líquido	líquido	NOUN	_	Gender=Masc|Number=Sing	55	nsubj	O	_
 44	em	em	ADP	_	_	46	case	O	_
 45	menor	pequeno	ADJ	_	Gender=Fem|Number=Sing	46	amod	O	_
 46	proporção	proporção	NOUN	_	Gender=Fem|Number=Sing	43	nmod	O	_
@@ -137659,12 +137659,12 @@
 48-49	no	_	_	_	_	_	_	_	_
 48	em	em	ADP	_	_	50	case	O	_
 49	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	50	det	O	_
-50	caso	caso	NOUN	_	Gender=Masc|Number=Sing	55	obl	O	_
+50	caso	caso	NOUN	_	Gender=Masc|Number=Sing	43	obl	O	_
 51	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	52	det	O	_
-52	água	água	NOUN	_	Gender=Fem|Number=Sing	55	nsubj	B=FLUIDO	_
-53	,	,	PUNCT	_	_	55	punct	O	_
-54	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	55	expl	O	_
-55	dispersa	disperso	ADJ	_	Gender=Fem|Number=Sing	43	amod	O	_
+52	água	água	NOUN	_	Gender=Fem|Number=Sing	50	nmod	B=FLUIDO	_
+53	,	,	PUNCT	_	_	50	punct	O	_
+54	se	se	PRON	_	_	55	expl	O	_
+55	dispersa	dispersar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 56-57	no	_	_	_	_	_	_	_	_
 56	em	em	ADP	_	_	58	case	O	_
 57	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	58	det	O	_
@@ -137676,7 +137676,7 @@
 63	diversos	diverso	DET	_	Gender=Masc|Number=Plur|PronType=Ind	64	det	O	_
 64	diâmetros	diâmetro	NOUN	_	Gender=Masc|Number=Plur	61	nmod	O	_
 65	.	.	PUNCT	_	_	55	punct	O	_
-66	.	.	PUNCT	_	_	16	punct	O	_
+66	.	.	PUNCT	_	_	55	punct	O	_
 
 # text = As gotas de diâmetros maiores tendem novamente a se aglutinar, no entanto, as de pequeno (entre 1 µm e 10 µm) não, permanecendo dispersas no meio oleoso, formando uma emulsão (BRASIL et al., 2012)..
 # sent_id = 202-20140404-MONOGRAFIA_0-20
@@ -145153,13 +145153,13 @@
 9	seja	ser	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	11	cop	O	_
 10	menos	menos	ADV	_	_	11	advmod	O	_
 11	frequente	frequente	ADJ	_	Gender=Fem|Number=Sing	32	advcl	O	_
-12	e	e	CCONJ	_	_	18	cc	O	_
-13	geralmente	geralmente	ADV	_	_	11	conj	O	_
+12	e	e	CCONJ	_	_	15	cc	O	_
+13	geralmente	geralmente	ADV	_	_	15	advmod	O	_
 14	de	de	ADP	_	_	15	case	O	_
 15	magnitudes	magnitude	NOUN	_	Gender=Fem|Number=Plur	11	conj	O	_
 16	menores	pequeno	ADJ	_	Gender=Fem|Number=Plur	15	amod	O	_
-17	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	18	expl	O	_
-18	comparadas	comparar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	15	acl	O	_
+17	se	se	SCONJ	_	_	18	mark	O	_
+18	comparadas	comparar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	15	advcl	O	_
 19-20	aos	_	_	_	_	_	_	_	_
 19	a	a	ADP	_	_	21	case	O	_
 20	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	21	det	O	_
@@ -145172,7 +145172,7 @@
 26	de	de	ADP	_	_	27	case	O	_
 27	placas	placa	NOUN	_	Gender=Fem|Number=Plur	25	nmod	O	_
 28	tectônicas	tectônico	ADJ	_	Gender=Fem|Number=Plur	27	amod	O	_
-29	,	,	PUNCT	_	_	22	punct	O	_
+29	,	,	PUNCT	_	_	18	punct	O	_
 30	não	não	ADV	_	Polarity=Neg	32	advmod	O	_
 31	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	32	expl	O	_
 32	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
@@ -147154,7 +147154,7 @@
 17	)	)	PUNCT	_	_	15	punct	O	_
 18	e	e	CCONJ	_	_	20	cc	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
-20	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	6	conj	B=BACIA:TIPO	_
+20	Bacia	bacia	PROPN	_	Gender=Fem|Number=Sing	2	conj	B=BACIA:TIPO	_
 21-22	do	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	20	flat:name	O	_
 22	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	20	flat:name	O	_
@@ -147163,8 +147163,8 @@
 25	consideradas	considerar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part|Voice=Pass	0	root	O	_
 26	relativamente	relativamente	ADV	_	_	25	advmod	O	_
 27	assísmicas	assísmico	ADJ	_	Gender=Fem|Number=Plur	25	xcomp	O	_
-28	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	29	expl	O	_
-29	comparadas	comparar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	27	advcl	O	_
+28	se	se	SCONJ	_	_	29	mark	O	_
+29	comparadas	comparar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	25	advcl	O	_
 30-31	às	_	_	_	_	_	_	_	_
 30	a	a	ADP	_	_	32	case	O	_
 31	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	32	det	O	_
@@ -148309,7 +148309,7 @@
 
 # text = C a p í t u l o 2 - S i s m i c i d a d e | 35 Nota-se que nos dados de mecanismo focais obtidos para o sismo São Vicente os mergulhos subvertical (89º) e sub-horizontal (3º) dos planos nodais diferem dos principais eventos registrados ao longo da margem continental sul e sudeste (45º) (Figura 2.5)..
 # sent_id = 258-20160602-TESEMSC_0-147
-1	C	C	PROPN	_	Gender=Masc|Number=Sing	3	parataxis	O	_
+1	C	C	PROPN	_	Gender=Masc|Number=Sing	3	parataxis	O	sentenciação
 2	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	3	det	O	_
 3	p	p	PROPN	_	Gender=Masc|Number=Sing	0	root	O	_
 4	í	í	PROPN	_	Number=Sing	3	flat:name	O	_
@@ -159349,8 +159349,8 @@
 6	refletor	refletor	NOUN	_	Gender=Masc|Number=Sing	4	obj	O	_
 7	mais	mais	ADV	_	_	8	advmod	O	_
 8	fraco	fraco	ADJ	_	Gender=Masc|Number=Sing	6	amod	O	_
-9	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	10	expl	O	_
-10	comparado	comparar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	6	acl	O	_
+9	se	se	SCONJ	_	_	10	mark	O	_
+10	comparado	comparar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	4	advcl	O	_
 11-12	aos	_	_	_	_	_	_	_	_
 11	a	a	ADP	_	_	13	case	O	_
 12	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	13	det	O	_
@@ -161337,8 +161337,8 @@
 8	de	de	ADP	_	_	10	case	O	_
 9	maior	grande	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
 10	elevação	elevação	NOUN	_	Gender=Fem|Number=Sing	7	nmod	O	_
-11	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	12	expl	O	_
-12	comparados	comparar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	7	acl	O	_
+11	se	se	SCONJ	_	_	12	mark	O	_
+12	comparados	comparar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	6	advcl	O	_
 13-14	às	_	_	_	_	_	_	_	_
 13	a	a	ADP	_	_	15	case	O	_
 14	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	15	det	O	_
@@ -229021,18 +229021,18 @@
 2	razão	razão	NOUN	_	Gender=Fem|Number=Sing	11	nsubj	O	_
 3	monóxido	monóxido	ADJ	_	Gender=Masc|Number=Sing	2	amod	O	_
 4	de	de	ADP	_	_	5	case	O	_
-5	carbono	carbono	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
-6	quimissorvido	quimissorver	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	2	acl	O	_
+5	carbono	carbono	NOUN	_	Gender=Masc|Number=Sing	3	nmod	O	_
+6	quimissorvido	quimissorver	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	3	acl	O	_
 7	por	por	ADP	_	_	8	case	O	_
-8	átomo	átomo	NOUN	_	Gender=Masc|Number=Sing	6	obl	O	_
+8	átomo	átomo	NOUN	_	Gender=Masc|Number=Sing	2	nmod	O	_
 9	de	de	ADP	_	_	10	case	O	_
 10	molibdênio	molibdênio	NOUN	_	Gender=Masc|Number=Sing	8	nmod	B=LITOLOGIA	_
 11	ficou	ficar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	O	_
 12	muito	muito	ADV	_	_	13	advmod	O	_
 13	abaixo	abaixo	ADV	_	_	11	advmod	O	_
 14	de	de	ADP	_	_	15	case	O	_
-15	um	um	NUM	_	NumType=Card	11	obl	O	_
-16	,	,	PUNCT	_	_	11	punct	O	_
+15	um	um	NUM	_	NumType=Card	13	obl	O	_
+16	,	,	PUNCT	_	_	20	punct	O	_
 17	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	20	nsubj	O	_
 18	seria	ser	AUX	_	Mood=Cnd|Number=Sing|Person=3|VerbForm=Fin	20	cop	O	_
 19	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	20	det	O	_
@@ -229047,9 +229047,9 @@
 28-29	das	_	_	_	_	_	_	_	_
 28	de	de	ADP	_	_	30	case	O	_
 29	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	30	det	O	_
-30	amostras	amostra	NOUN	_	Gender=Fem|Number=Plur	27	nmod	O	_
+30	amostras	amostra	NOUN	_	Gender=Fem|Number=Plur	25	nmod	O	_
 31	estivessem	estar	AUX	_	Mood=Sub|Number=Plur|Person=3|Tense=Imp|VerbForm=Fin	32	cop	O	_
-32	disponíveis	disponível	ADJ	_	Gender=Fem|Number=Plur	20	acl:relcl	O	_
+32	disponíveis	disponível	ADJ	_	Gender=Fem|Number=Plur	20	advcl	O	_
 33	para	para	ADP	_	_	34	case	O	_
 34	quimissorção	quimissorção	NOUN	_	Gender=Fem|Number=Sing	32	obl	B=QUÍMICA	_
 35	.	.	PUNCT	_	_	11	punct	O	_
@@ -236510,15 +236510,15 @@
 21	superfícies	superfície	NOUN	_	Gender=Fem|Number=Plur	19	obl:agent	O	_
 22	de	de	ADP	_	_	23	case	O	_
 23	erosão	erosão	NOUN	_	Gender=Fem|Number=Sing	21	nmod	O	_
-24	,	,	PUNCT	_	_	19	punct	O	_
+24	,	,	PUNCT	_	_	26	punct	O	_
 25	as	o	DET	_	Gender=Fem|Number=Plur|PronType=Art	26	det	O	_
 26	quais	qual	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	32	nsubj	O	_
 27	,	,	PUNCT	_	_	30	punct	O	_
-28	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Plur|Person=3|PronType=Prs	30	expl	O	_
+28	se	se	SCONJ	_	_	30	mark	O	_
 29	corretamente	corretamente	ADV	_	_	30	advmod	O	_
-30	correlacionadas	correlacionar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	26	conj	O	_
+30	correlacionadas	correlacionar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	32	advcl	O	_
 31	,	,	PUNCT	_	_	30	punct	O	_
-32	servem	servir	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	18	acl:relcl	O	_
+32	servem	servir	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	21	acl:relcl	O	_
 33	de	de	ADP	_	_	35	case	O	_
 34	excelentes	excelente	ADJ	_	Gender=Masc|Number=Plur	35	amod	O	_
 35	marcos	marco	NOUN	_	Gender=Masc|Number=Plur	32	obj	O	_
@@ -254169,11 +254169,11 @@
 2-3	desta	_	_	_	_	_	_	_	_
 2	de	de	ADP	_	_	1	fixed	O	_
 3	esta	este	DET	_	Gender=Fem|Number=Sing|PronType=Dem	4	det	O	_
-4	informação	informação	NOUN	_	Gender=Fem|Number=Sing	10	conj	O	_
+4	informação	informação	NOUN	_	Gender=Fem|Number=Sing	19	conj	O	_
 5	escassa	escasso	ADJ	_	Gender=Fem|Number=Sing	4	amod	O	_
 6	,	,	PUNCT	_	_	4	punct	O	_
-7	se	se	SCONJ	_	_	10	mark	O	_
-8	for	ser	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	10	cop	O	_
+7	se	se	SCONJ	_	_	9	mark	O	_
+8	for	ser	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Fut|VerbForm=Fin	9	cop	O	_
 9	disponível	disponível	ADJ	_	Gender=Fem|Number=Sing	19	advcl	O	_
 10	informação	informação	NOUN	_	Gender=Fem|Number=Sing	9	nsubj	O	_
 11	extra	extra	ADJ	_	Gender=Fem|Number=Sing	10	amod	O	_
@@ -254184,7 +254184,7 @@
 15	de	de	ADP	_	_	17	case	O	_
 16	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
 17	fonte	fonte	NOUN	_	Gender=Fem|Number=Sing	14	nmod	O	_
-18	,	,	PUNCT	_	_	10	punct	O	_
+18	,	,	PUNCT	_	_	9	punct	O	_
 19-20	pode-se	_	_	_	_	_	_	_	_
 19	pode	poder	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 20	se	se	PRON	_	Case=Acc|Gender=Unsp|PronType=Prs	19	expl	O	_
@@ -254213,7 +254213,7 @@
 42	aumenta	aumentar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	28	advcl	O	_
 43	e	e	CCONJ	_	_	45	cc	O	_
 44	novos	novo	ADJ	_	Gender=Masc|Number=Plur	45	amod	O	_
-45	elementos	elemento	NOUN	_	Gender=Masc|Number=Plur	42	obj	O	_
+45	elementos	elemento	NOUN	_	Gender=Masc|Number=Plur	48	nsubj:pass	O	_
 46	geométricos	geométrico	ADJ	_	Gender=Masc|Number=Plur	45	amod	O	_
 47	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	48	aux:pass	O	_
 48	acrescentados	acrescentar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	42	conj	O	_
@@ -255763,8 +255763,8 @@
 
 # text = De acordo com o que é sugerido e utilizando o método de tentativa e erro é preenchida a porção abaixo da reta superficial com outras retas horizontais empilhadas de forma a simular um corpo, de contraste negativo de susceptibilidade, que se estende e se estreita em profundidade sob a forma de uma calha triangular (figura 4.12)..
 # sent_id = 20-20140904-TESEDSC_0-609
-1	De	de	ADP	_	_	2	case	O	_
-2	acordo	acordo	NOUN	_	Gender=Masc|Number=Sing	17	obl	O	_
+1	De	de	ADP	_	_	4	case	O	_
+2	acordo	acordo	NOUN	_	Gender=Masc|Number=Sing	1	fixed	O	_
 3	com	com	ADP	_	_	1	fixed	O	_
 4	o	o	PRON	_	Gender=Masc|Number=Sing|PronType=Dem	17	obl	O	_
 5	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	7	nsubj:pass	O	_
@@ -255786,17 +255786,17 @@
 21-22	da	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	23	case	O	_
 22	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	23	det	O	_
-23	reta	reta	NOUN	_	Gender=Fem|Number=Sing	19	nmod	O	_
+23	reta	reta	NOUN	_	Gender=Fem|Number=Sing	20	obl	O	_
 24	superficial	superficial	ADJ	_	Gender=Fem|Number=Sing	23	amod	O	_
 25	com	com	ADP	_	_	27	case	O	_
 26	outras	outro	DET	_	Gender=Fem|Number=Plur|PronType=Ind	27	det	O	_
-27	retas	reta	NOUN	_	Gender=Fem|Number=Plur	23	nmod	O	_
+27	retas	reta	NOUN	_	Gender=Fem|Number=Plur	17	obl	O	_
 28	horizontais	horizontal	ADJ	_	Gender=Fem|Number=Plur	27	amod	O	_
 29	empilhadas	empilhar	VERB	_	Gender=Fem|Number=Plur|VerbForm=Part	27	acl	O	_
-30	de	de	ADP	_	_	31	case	O	_
-31	forma	forma	NOUN	_	Gender=Fem|Number=Sing	27	nmod	O	_
-32	a	a	SCONJ	_	_	33	mark	O	_
-33	simular	simular	VERB	_	VerbForm=Inf	29	advcl	O	_
+30	de	de	ADP	_	_	33	mark	O	MWEPOS=SCONJ
+31	forma	forma	NOUN	_	Gender=Fem|Number=Sing	30	fixed	O	_
+32	a	a	SCONJ	_	_	30	fixed	O	_
+33	simular	simular	VERB	_	VerbForm=Inf	17	advcl	O	_
 34	um	um	DET	_	Definite=Ind|Gender=Masc|Number=Sing|PronType=Art	35	det	O	_
 35	corpo	corpo	NOUN	_	Gender=Masc|Number=Sing	33	obj	O	_
 36	,	,	PUNCT	_	_	38	punct	O	_
@@ -255810,8 +255810,8 @@
 44	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	45	expl	O	_
 45	estende	estender	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	35	acl:relcl	O	_
 46	e	e	CCONJ	_	_	48	cc	O	_
-47	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	48	expl	O	_
-48	estreita	estreito	ADJ	_	Gender=Fem|Number=Sing	45	conj	B=LOCAL:GEO	_
+47	se	se	PRON	_	_	48	expl	O	_
+48	estreita	estreitar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	45	conj	B=LOCAL:GEO	_
 49	em	em	ADP	_	_	50	case	O	_
 50	profundidade	profundidade	NOUN	_	Gender=Fem|Number=Sing	48	obl	O	_
 51	sob	sob	ADP	_	_	53	case	O	_
@@ -268847,33 +268847,33 @@
 3	poderiam	poder	VERB	_	Mood=Cnd|Number=Plur|Person=3|VerbForm=Fin	0	root	O	_
 4	ser	ser	AUX	_	VerbForm=Inf	5	aux:pass	O	_
 5	explicados	explicar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part|Voice=Pass	3	xcomp	O	_
-6	,	,	PUNCT	_	_	5	punct	O	_
+6	,	,	PUNCT	_	_	9	punct	O	_
 7-8	pelo	_	_	_	_	_	_	_	_
 7	por	por	ADP	_	_	9	case	O	_
 8	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	9	det	O	_
-9	menos	menos	NOUN	_	Gender=Masc|Number=Plur	5	obl:agent	O	_
+9	menos	menos	NOUN	_	Gender=Masc|Number=Plur	5	obl	O	_
 10-11	na	_	_	_	_	_	_	_	_
 10	em	em	ADP	_	_	12	case	O	_
 11	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	12	det	O	_
-12	porção	porção	NOUN	_	Gender=Fem|Number=Sing	5	obl	O	_
+12	porção	porção	NOUN	_	Gender=Fem|Number=Sing	9	obl	O	_
 13	superficial	superficial	ADJ	_	Gender=Fem|Number=Sing	12	amod	O	_
 14-15	dos	_	_	_	_	_	_	_	_
 14	de	de	ADP	_	_	16	case	O	_
 15	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	16	det	O	_
 16	modelos	modelo	NOUN	_	Gender=Masc|Number=Plur	12	nmod	O	_
-17	,	,	PUNCT	_	_	5	punct	O	_
+17	,	,	PUNCT	_	_	9	punct	O	_
 18-19	pela	_	_	_	_	_	_	_	_
 18	por	por	ADP	_	_	20	case	O	_
 19	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	20	det	O	_
-20	presença	presença	NOUN	_	Gender=Fem|Number=Sing	5	obl	O	_
+20	presença	presença	NOUN	_	Gender=Fem|Number=Sing	5	obl:agent	O	_
 21-22	do	_	_	_	_	_	_	_	_
 21	de	de	ADP	_	_	23	case	O	_
 22	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	23	det	O	_
 23	pacote	pacote	NOUN	_	Gender=Masc|Number=Sing	20	nmod	O	_
 24	sedimentar	sedimentar	ADJ	_	Gender=Masc|Number=Sing	23	amod	O	_
-25	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	27	nsubj	O	_
-26	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	27	expl	O	_
-27	espessa	espesso	ADJ	_	Gender=Fem|Number=Sing	23	acl:relcl	O	_
+25	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	27	nsubj:pass	O	_
+26	se	se	PRON	_	_	27	expl	O	_
+27	espessa	espessar	VERB	_	Mood=Ind|Number=Sing|Person=3|VerbForm=Fin|Voice=Pass	23	acl:relcl	O	_
 28	suavemente	suavemente	ADV	_	_	27	advmod	O	_
 29	para	para	ADP	_	_	30	case	O	_
 30	SE..	SE..	PROPN	_	Gender=Masc|Number=Sing	27	obl	O	_
@@ -268887,7 +268887,7 @@
 4	HH	HH	PROPN	_	Gender=Masc|Number=Sing	3	nmod	O	_
 5	’	’	PROPN	_	Number=Sing	4	flat:name	O	_
 6	(	(	PUNCT	_	_	7	punct	O	_
-7	figura	figura	NOUN	_	Gender=Fem|Number=Sing	14	parataxis	O	_
+7	figura	figura	NOUN	_	Gender=Fem|Number=Sing	3	parataxis	O	_
 8	6.16	6.16	NUM	_	NumType=Card	7	nummod	O	_
 9	)	)	PUNCT	_	_	7	punct	O	_
 10	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	11	det	O	_
@@ -268898,17 +268898,17 @@
 15	por	por	ADP	_	_	17	case	O	_
 16	uma	um	DET	_	Definite=Ind|Gender=Fem|Number=Sing|PronType=Art	17	det	O	_
 17	cunha	cunha	NOUN	_	Gender=Fem|Number=Sing	14	obl:agent	O	_
-18	,	,	PUNCT	_	_	14	punct	O	_
+18	,	,	PUNCT	_	_	21	punct	O	_
 19-20	no	_	_	_	_	_	_	_	_
 19	em	em	ADP	_	_	21	case	O	_
 20	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	21	det	O	_
-21	tom	tom	NOUN	_	Gender=Masc|Number=Sing	14	obl	O	_
+21	tom	tom	NOUN	_	Gender=Masc|Number=Sing	17	obl	O	_
 22	azul	azul	ADJ	_	Gender=Masc|Number=Sing	21	amod	O	_
 23	escuro	escuro	ADJ	_	Gender=Masc|Number=Sing	21	amod	O	_
-24	,	,	PUNCT	_	_	14	punct	O	_
-25	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	27	nsubj	O	_
-26	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	27	expl	O	_
-27	espessa	espesso	ADJ	_	Gender=Fem|Number=Sing	21	acl:relcl	O	_
+24	,	,	PUNCT	_	_	21	punct	O	_
+25	que	que	PRON	_	Gender=Masc|Number=Sing|PronType=Rel	27	nsubj:pass	O	_
+26	se	se	PRON	_	_	27	expl	O	_
+27	espessa	espessar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin|Voice=Pass	17	acl:relcl	O	_
 28	para	para	ADP	_	_	29	case	O	_
 29	sudeste	sudeste	NOUN	_	Gender=Masc|Number=Sing	27	obl	O	_
 30-31	do	_	_	_	_	_	_	_	_
@@ -283552,9 +283552,9 @@
 20	rifte	rifte	ADJ	_	Gender=Fem|Number=Sing	19	amod	B=LOCAL:GEO	_
 21	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	22	expl	O	_
 22	aproveitaram	aproveitar	VERB	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	0	root	O	_
-23	,	,	PUNCT	_	_	22	punct	O	_
+23	,	,	PUNCT	_	_	24	punct	O	_
 24	possivelmente	possivelmente	ADV	_	_	22	advmod	O	_
-25	,	,	PUNCT	_	_	22	punct	O	_
+25	,	,	PUNCT	_	_	24	punct	O	_
 26-27	do	_	_	_	_	_	_	_	_
 26	de	de	ADP	_	_	28	case	O	_
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
@@ -283570,13 +283570,13 @@
 36	e	e	CCONJ	_	_	37	cc	O	_
 37	Cabo	Cabo	PROPN	_	Gender=Masc|Number=Sing	35	conj	O	_
 38	Frio	Frio	PROPN	_	Number=Sing	37	flat:name	O	_
-39	,	,	PUNCT	_	_	41	punct	O	_
+39	,	,	PUNCT	_	_	34	punct	O	_
 40	de	de	ADP	_	_	41	case	O	_
 41	direção	direção	NOUN	_	Gender=Fem|Number=Sing	28	nmod	O	_
 42	NE-SW	NE-SW	PROPN	_	Gender=Fem|Number=Sing	41	nmod	O	_
 43	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	45	nsubj	O	_
-44	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	45	expl	O	_
-45	expressa	esprimir	VERB	_	Number=Sing|VerbForm=Part	41	acl:relcl	O	_
+44	se	se	PRON	_	_	45	expl	O	_
+45	expressa	expressar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	28	acl:relcl	O	_
 46	claramente	claramente	ADV	_	_	45	advmod	O	_
 47-48	no	_	_	_	_	_	_	_	_
 47	em	em	ADP	_	_	49	case	O	_
@@ -291322,7 +291322,7 @@
 7	são	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	9	cop	O	_
 8	muito	muito	ADV	_	_	9	advmod	O	_
 9	caros	caro	ADJ	_	Gender=Masc|Number=Plur	0	root	O	_
-10	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	9	expl	O	_
+10	se	se	SCONJ	_	_	11	mark	O	_
 11	comparados	comparar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	9	advcl	O	_
 12	com	com	ADP	_	_	13	case	O	_
 13	óleo	óleo	NOUN	_	Gender=Masc|Number=Sing	11	obl	B=PRODUTO|B=FLUIDO	_
@@ -294151,8 +294151,8 @@
 12	sódica	sódico	ADJ	_	Gender=Fem|Number=Sing	11	amod	O	_
 13	incha	inchar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	1	ccomp	O	_
 14	mais	mais	ADV	_	_	16	advmod	O	_
-15	se	se	PRON	_	Case=Acc|Gender=Fem|Number=Sing|Person=3|PronType=Prs	16	expl	O	_
-16	comparada	comparar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	13	xcomp	O	_
+15	se	se	SCONJ	_	_	16	mark	O	_
+16	comparada	comparar	VERB	_	Gender=Fem|Number=Sing|VerbForm=Part	13	advcl	O	_
 17	com	com	ADP	_	_	19	case	O	_
 18	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	19	det	O	_
 19	cálcica	cálcica	NOUN	_	Gender=Fem|Number=Sing	16	obl	O	_
@@ -307492,7 +307492,7 @@
 21	(	(	PUNCT	_	_	22	punct	O	_
 22	a	ela	PRON	_	Gender=Fem|Number=Sing|PronType=Dem	19	appos	O	_
 23	)	)	PUNCT	_	_	22	punct	O	_
-24	,	,	PUNCT	_	_	16	punct	O	_
+24	,	,	PUNCT	_	_	28	punct	O	_
 25	pois	pois	SCONJ	_	_	28	mark	O	_
 26	este	este	DET	_	Gender=Masc|Number=Sing|PronType=Dem	27	det	O	_
 27	experimento	experimento	NOUN	_	Gender=Masc|Number=Sing	28	nsubj	O	_
@@ -307505,8 +307505,8 @@
 34	seu	seu	DET	_	Gender=Masc|Number=Sing|PronType=Prs	35	det	O	_
 35	encerramento	encerramento	NOUN	_	Gender=Masc|Number=Sing	28	obl	O	_
 36	,	,	PUNCT	_	_	38	punct	O	_
-37	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	38	expl	O	_
-38	comparado	comparar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	35	acl	O	_
+37	se	se	SCONJ	_	_	38	mark	O	_
+38	comparado	comparar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	28	advcl	O	_
 39	com	com	ADP	_	_	42	case	O	_
 40	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	42	det	O	_
 41	outros	outro	DET	_	Gender=Masc|Number=Plur|PronType=Ind	42	det	O	_
@@ -308404,8 +308404,8 @@
 24	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	7	obl	O	_
 25	sem	sem	ADP	_	_	26	case	O	_
 26	argila	argila	NOUN	_	Gender=Fem|Number=Sing	24	nmod	B=LITOLOGIA	_
-27	,	,	PUNCT	_	_	7	punct	O	_
-28	pois	pois	SCONJ	_	_	43	mark	O	_
+27	,	,	PUNCT	_	_	28	punct	O	_
+28	pois	pois	CCONJ	_	_	43	cc	O	_
 29	,	,	PUNCT	_	_	28	punct	O	_
 30	os	o	DET	_	Definite=Def|Gender=Masc|Number=Plur|PronType=Art	31	det	O	_
 31	processos	processo	NOUN	_	Gender=Masc|Number=Plur	43	nsubj	O	_
@@ -308421,8 +308421,8 @@
 40	procedimento	procedimento	NOUN	_	Gender=Masc|Number=Sing	36	nmod	O	_
 41	foram	ser	AUX	_	Mood=Ind|Number=Plur|Person=3|VerbForm=Fin	43	cop	O	_
 42	mais	mais	ADV	_	_	43	advmod	O	_
-43	rápidos	rápido	ADJ	_	Gender=Masc|Number=Plur	7	advcl	O	_
-44	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Plur|Person=3|PronType=Prs	43	expl	O	_
+43	rápidos	rápido	ADJ	_	Gender=Masc|Number=Plur	7	conj	O	_
+44	se	se	SCONJ	_	_	45	mark	O	_
 45	comparados	comparar	VERB	_	Gender=Masc|Number=Plur|VerbForm=Part	43	advcl	O	_
 46	com	com	ADP	_	_	47	case	O	_
 47	os	o	PRON	_	Gender=Masc|Number=Plur|PronType=Dem	45	obl	O	_
@@ -308916,7 +308916,7 @@
 18	visto	ver	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	50	mark	O	MWEPOS=SCONJ
 19	que	que	SCONJ	_	_	18	fixed	O	_
 20	,	,	PUNCT	_	_	22	punct	O	_
-21	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	22	expl	O	_
+21	se	se	SCONJ	_	_	22	mark	O	_
 22	comparado	comparar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	50	advcl	O	_
 23	com	com	ADP	_	_	25	case	O	_
 24	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	25	det	O	_
@@ -308944,7 +308944,7 @@
 45	(	(	PUNCT	_	_	46	punct	O	_
 46	NaI	NaI	PROPN	_	Gender=Masc|Number=Sing	42	appos	O	_
 47	)	)	PUNCT	_	_	46	punct	O	_
-48	,	,	PUNCT	_	_	40	punct	O	_
+48	,	,	PUNCT	_	_	33	punct	O	_
 49	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	50	aux:pass	O	_
 50	produzido	produzir	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part|Voice=Pass	4	advcl	O	_
 51	.	.	PUNCT	_	_	1	punct	O	_
@@ -318016,7 +318016,7 @@
 15	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	16	det	O	_
 16	água	água	NOUN	_	Gender=Fem|Number=Sing	13	obl	B=FLUIDO	_
 17	(	(	PUNCT	_	_	18	punct	O	_
-18	emulsão	emulsão	NOUN	_	Gender=Fem|Number=Sing	16	appos	B=QUÍMICA	_
+18	emulsão	emulsão	NOUN	_	Gender=Fem|Number=Sing	13	appos	B=QUÍMICA	_
 19	direta	direto	ADJ	_	Gender=Fem|Number=Sing	18	amod	O	_
 20	)	)	PUNCT	_	_	18	punct	O	_
 21	ou	ou	CCONJ	_	_	26	cc	O	_
@@ -318024,7 +318024,7 @@
 23	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	24	det	O	_
 24	água	água	NOUN	_	Gender=Fem|Number=Sing	26	nsubj	B=FLUIDO	_
 25	está	estar	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	26	cop	O	_
-26	emulsionada	emulsionado	ADJ	_	Gender=Fem|Number=Sing	8	conj	O	_
+26	emulsionada	emulsionado	ADJ	_	Gender=Fem|Number=Sing	13	conj	O	_
 27-28	no	_	_	_	_	_	_	_	_
 27	em	em	ADP	_	_	29	case	O	_
 28	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	29	det	O	_
@@ -320515,18 +320515,18 @@
 11	como	como	ADP	_	_	13	case	O	_
 12	a	o	DET	_	Definite=Def|Gender=Fem|Number=Sing|PronType=Art	13	det	O	_
 13	viscosidade	viscosidade	NOUN	_	Gender=Fem|Number=Sing	5	xcomp	O	_
-14	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	20	nsubj	O	_
+14	que	que	PRON	_	Gender=Fem|Number=Sing|PronType=Rel	17	nsubj	O	_
 15	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	16	det	O	_
-16	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	20	nsubj	O	_
-17	teria	ter	AUX	_	Mood=Cnd|Number=Sing|Person=3|VerbForm=Fin	20	aux	O	_
+16	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	17	nsubj	O	_
+17	teria	ter	VERB	_	Mood=Cnd|Number=Sing|Person=3|VerbForm=Fin	13	acl:relcl	O	_
 18	se	se	SCONJ	_	_	20	mark	O	_
 19	fosse	ser	AUX	_	Mood=Sub|Number=Sing|Person=3|Tense=Imp|VerbForm=Fin	20	cop	O	_
-20	Newtoniano	newtoniano	ADJ	_	Gender=Masc|Number=Sing	13	acl:relcl	O	_
-21	,	,	PUNCT	_	_	20	punct	O	_
+20	Newtoniano	newtoniano	ADJ	_	Gender=Masc|Number=Sing	17	advcl	O	_
+21	,	,	PUNCT	_	_	24	punct	O	_
 22-23	naquelas	_	_	_	_	_	_	_	_
 22	em	em	ADP	_	_	24	case	O	_
 23	aquelas	aquele	DET	_	Gender=Fem|Number=Plur|PronType=Dem	24	det	O	_
-24	condições	condição	NOUN	_	Gender=Fem|Number=Plur	20	obl	O	_
+24	condições	condição	NOUN	_	Gender=Fem|Number=Plur	17	obl	O	_
 25	de	de	ADP	_	_	26	case	O	_
 26	fluxo	fluxo	NOUN	_	Gender=Masc|Number=Sing	24	nmod	O	_
 27	(	(	PUNCT	_	_	28	punct	O	_
@@ -320553,7 +320553,7 @@
 48-49	ao	_	_	_	_	_	_	_	_
 48	a	a	ADP	_	_	50	case	O	_
 49	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	50	det	O	_
-50	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	47	obj	O	_
+50	fluido	fluido	NOUN	_	Gender=Masc|Number=Sing	47	obl	O	_
 51	para	para	SCONJ	_	_	53	mark	O	_
 52	que	que	SCONJ	_	_	53	mark	O	_
 53	haja	haver	VERB	_	Mood=Sub|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	47	advcl	O	_
@@ -323755,16 +323755,16 @@
 5	retrata	retratar	VERB	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	O	_
 6	em	em	ADP	_	_	8	case	O	_
 7	que	que	DET	_	Gender=Fem|Number=Sing|PronType=Int	8	det	O	_
-8	condição	condição	NOUN	_	Gender=Fem|Number=Sing	5	obj	O	_
+8	condição	condição	NOUN	_	Gender=Fem|Number=Sing	11	obl	O	_
 9	experimental	experimental	ADJ	_	Gender=Fem|Number=Sing	8	amod	O	_
 10	é	ser	AUX	_	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	11	cop	O	_
-11	possível	possível	ADJ	_	Gender=Fem|Number=Sing	5	parataxis	O	_
-12	se	se	PRON	_	_	11	expl	O	_
+11	possível	possível	ADJ	_	Gender=Fem|Number=Sing	5	ccomp	O	_
+12	se	se	PRON	_	_	13	expl	O	_
 13	obter	obter	VERB	_	VerbForm=Inf	11	csubj	O	_
 14	microemulsões	microemulsão	NOUN	_	Gender=Fem|Number=Plur	13	obj	O	_
 15	e	e	CCONJ	_	_	17	cc	O	_
 16	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	17	det	O	_
-17	regiões	região	NOUN	_	Gender=Fem|Number=Plur	14	conj	O	_
+17	regiões	região	NOUN	_	Gender=Fem|Number=Plur	5	conj	O	_
 18	que	que	PRON	_	Gender=Fem|Number=Plur|PronType=Rel	19	nsubj	O	_
 19	limitam	limitar	VERB	_	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	17	acl:relcl	O	_
 20	as	o	DET	_	Definite=Def|Gender=Fem|Number=Plur|PronType=Art	21	det	O	_
@@ -328882,8 +328882,8 @@
 21	tensoativo	tensoativo	NOUN	_	Gender=Masc|Number=Sing	18	nmod	O	_
 22	Ultranex	Ultranex	PROPN	_	Gender=Masc|Number=Sing	21	nmod	O	_
 23	40	40	PROPN	_	Number=Sing	22	flat:name	O	_
-24	se	se	PRON	_	Case=Acc|Gender=Masc|Number=Sing|Person=3|PronType=Prs	25	expl	O	_
-25	comparado	comparar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	21	acl	O	_
+24	se	se	SCONJ	_	_	25	mark	O	_
+25	comparado	comparar	VERB	_	Gender=Masc|Number=Sing|VerbForm=Part	15	advcl	O	_
 26-27	ao	_	_	_	_	_	_	_	_
 26	a	a	ADP	_	_	28	case	O	_
 27	o	o	DET	_	Definite=Def|Gender=Masc|Number=Sing|PronType=Art	28	det	O	_
